cognite/neat/_graph/transformers/_value_type.py

```diff
@@ -64,3 +71,73 @@ def transform(self, graph: Graph) -> None:
                     graph.remove((s, property_uri, o))
                     new_property = URIRef(f"{property_uri}_{remove_namespace_from_uri(value_type_uri)}")
                     graph.add((s, new_property, o))
+
+
+class ConvertLiteral(BaseTransformer):
+    description: str = "ConvertLiteral is a transformer that converts a literal value."
+    _use_only_once: bool = False
+    _need_changes = frozenset({})
+
+    _count_by_properties = """SELECT (COUNT(?property) AS ?propertyCount)
+    WHERE {{
+      ?instance a <{subject_type}> .
+      ?instance <{subject_predicate}> ?property
+    }}"""
+
+    _properties = """SELECT ?instance ?property
+    WHERE {{
+      ?instance a <{subject_type}> .
+      ?instance <{subject_predicate}> ?property
+    }}"""
```

Comment:
This will now only return you values which are literals
```suggestion
    _properties = """SELECT ?instance ?value
    WHERE {{
      ?instance a <{subject_type}> .
      ?instance <{subject_predicate}> ?value
      
      FILTER(isIRI(?value))
      
    }}"""
```